features/headings.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
-name: <h1>
-description: The `<h1>` element TODO.
+name: Section headings elements
+description: The `<h1>` through `<h6>` elements represent six levels of section headings.
 spec:
```

Comment:
There's just a single section in the spec here, with many IDs. Let's use just https://html.spec.whatwg.org/multipage/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements, which is what the ToC links to.

Reply:
done